src/bin/info.rs

```diff
@@ -17,13 +18,17 @@ async fn main() {
     funding_history_example(&info_client).await;
     l2_snapshot_example(&info_client).await;
     candles_snapshot_example(&info_client).await;
+    user_token_balances_example(&info_client).await;
+    user_fees_example(&info_client).await;
+    user_funding_example(&info_client).await;
+    spot_meta_example(&info_client).await;
+    spot_meta_and_asset_contexts_example(&info_client).await;
+    query_order_by_oid_example(&info_client).await;
+    query_referral_state_example(&info_client).await;
 }
 
 async fn open_orders_example(info_client: &InfoClient) {
-    let user: String = "0xc64cc00b46101bd40aa1c3121195e85c0b0918d8"
-        .parse()
-        .unwrap();
-    let user = H160::from_str(&user).unwrap();
+    let user = ADDRESS.to_owned().parse::<H160>().unwrap();
```

Comment:
nit: just define a function address and use that everywhere to further reduce copy-paste, i.e.
```
fn address() -> H160 {
  "0xc64cc00b46101bd40aa1c3121195e85c0b0918d8".to_string().parse().unwrap()
}
```

Reply:
Done